src/backend/distributed/citus--7.5-5--7.5-6.sql

```diff
@@ -0,0 +1,35 @@
+/* citus--7.5-1--7.5-2 */
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION pg_catalog.poolinfo_valid(text)
+	RETURNS boolean
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$poolinfo_valid$$;
+COMMENT ON FUNCTION poolinfo_valid(text) IS 'returns whether a poolinfo is valid';
+
+CREATE TABLE citus.pg_dist_poolinfo (
+    nodeid integer PRIMARY KEY
+                   REFERENCES pg_dist_node(nodeid)
+                              ON DELETE CASCADE,
```

Comment:
Not relevant to this PR, but just to make note of it: There's a good chance that `master_remove_node` does not invoke triggers.